### PR TITLE
Add NUbook troubleshooting page

### PR DESCRIPTION
**Pull request by JosephusPaye**
Add a troubleshooting page for NUbook, for help with common errors.

[Preview](https://deploy-preview-48--nubook.netlify.app/guides/nubook/troubleshooting)

### CODE REVIEW

File: src/book/03-guides/04-nubook/01-troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## My newly added page is not showing up
+
+When you add a new page, you should remove the gatsby build `cache` and `public` folder:
+
+```sh
+rm -rf .cache public
+```
+
+## I'm getting wierd errors when running `yarn dev`
```
Comment (ysims) [inline src/book/03-guides/04-nubook/01-troubleshooting.mdx:17]:
```suggestion
## I'm getting weird errors when running `yarn dev`
```

File: src/book/03-guides/04-nubook/01-troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+section: Guides
+chapter: NUbook
+title: Troubleshooting
+description: Learn how to troubleshoot common errors when working with NUbook.
+slug: /guides/nubook/troubleshooting
+---
+
+## My newly added page is not showing up
+
+When you add a new page, you should remove the gatsby build `cache` and `public` folder:
```
Comment (ysims) [inline src/book/03-guides/04-nubook/01-troubleshooting.mdx:11]:
```suggestion
When you add a new page, you should remove the gatsby build `.cache` and `public` folder:
```

**Comment by Bidski** [comment]
Is it possible to add a TOC to the top of the page? The one thats off to the right cuts off half of the question, having a TOC at the top of that would allow you to see the entirety of the question and make navigation easier

image

File: src/book/03-guides/04-nubook/01-troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+section: Guides
+chapter: NUbook
+title: Troubleshooting
+description: Learn how to troubleshoot common errors when working with NUbook.
+slug: /guides/nubook/troubleshooting
+---
+
+## My newly added page is not showing up
```
Comment (Bidski) [inline src/book/03-guides/04-nubook/01-troubleshooting.mdx:9]:
```suggestion
## Why is my newly added page is not showing up?
```

File: src/book/03-guides/04-nubook/01-troubleshooting.mdx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+When you add a new page, you should remove the gatsby build `.cache` and `public` folder:
+
+```sh
+rm -rf .cache public
+```
+
+## I'm getting weird errors when running `yarn dev`
```
Comment (Bidski) [inline src/book/03-guides/04-nubook/01-troubleshooting.mdx:17]:
```suggestion
## Why am I getting weird errors when I run `yarn dev`?
```

**Comment by JosephusPaye** [comment]
> Is it possible to add a TOC to the top of the page? The one thats off to the right cuts off half of the question, having a TOC at the top of that would allow you to see the entirety of the question and make navigation easier [...]

Make sense, we could start it at the top, and make it appear on the side when you scroll down away from it.

File: src/book/03-guides/04-nubook/01-troubleshooting.mdx (hunk 3 above)
Comment (JosephusPaye) [reply src/book/03-guides/04-nubook/01-troubleshooting.mdx:9]: I like this question format better 👍

**Comment by ysims** [comment]
Can this be merged now then? Alex's comment is probably something for another PR.

**Pull request merged**